### PR TITLE
Continuous int87

### PR DESCRIPTION
**Pull request by WalrusArtist**

### CODE REVIEW

**Review by platisd — DISMISSED** [review]
I wouldn't do it that way but I guess there are different ways to it. Let's see.

**Comment by WalrusArtist** [comment]
I seem to have the parsing problem once again with my last commit, ill resolve that, and then see if it works. If not, ill go with the way your suggested.

File: .travis.yml [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
+    script:
+    - build_platform mega2560
+    cache:
+      directories:
+      - "~/arduino_ide"
+      - "~/.arduino15/packages/"
 deploy:
   provider: releases
-  api_key:
-    secure: "EWEbN6iiVp0zKqkJJuqxu0OMlJlb2hLdYMfUDAu6zpeY58DI/C7thQFJoea+xhCpfKUybxktQpggmljLw1yf9a0d2qr9+LkniVWDR+N8pmgPZmksFTCoWtqoBYp4X6hG/2rUSrL6vhnZXsgOsVWlEZ77WJUXaum1mglLrPaqRntNX4YwSsG5cb+967lAw0BEz+V9eIdrhN2tVRpR49uDoo5sywLbmoqfGRn+TYqtfZFvuM549pWc1gWhV0y7IbQMeAw5FFiF9NHVHpwOhMW7qcANSH5XQbSX/U06avrFD8jtQriFgk7deZkIhsNcHgULXv6J85o2PG//GpZOUXVDhdPEG96R4nUFajrPtRXeCKccD1P9zFks1fIwKrsFRMAkm/XudvDaG73eTguIVOIQ0qSK8Xj49ch+yDoNOXac3oqFT/atFI+p0Z1RGbi9BnKMc6lIkfYraLwNQL+pBaxPREoJKPB89ha3+aVcJaR3QwBOgUMndEWVxMaGZC00OTzjF7VCqDuHq6rr400zjPsE759ATH1eGcuqcaZaWPG568c64IbLkuZvmi3HfxL1ozKTjCRABh7a50axx0QfnP11hgLJ1akO7h2Oh+RSUS2mBBwH55rPNf6jtZ/gE1kgyCHShUFX7bAQW8EGYjzy62fpT5z6biHJ1Tn75eaRb0O0TYA="
```
Comment (platisd) [inline .travis.yml:29]: You shouldn't remove this completely. Instead you should use the variable name that you used during encryption.

**Review by platisd — APPROVED** [review]
Let's see :crossed_fingers:

**Pull request merged**